pkg/nodejs/prepare/prepare.go

```diff
@@ -150,21 +150,37 @@ func addProbes(hasNodejsApplication bool, writer util.FileWriter) error {
 	if err != nil {
 		return err
 	}
+	err = os.Chmod(buildPath+"/overrides/liveness_node.sh", 0755)
+	if err != nil {
+		return errors.Wrap(err, "Could not set file permissions")
+	}
 	err = writer(util.NewTemplateWriter(nginxProbe, "nginxliveness", ReadinessLivenessSH),
 		"overrides", "liveness_nginx.sh")
 	if err != nil {
 		return err
 	}
+	err = os.Chmod(buildPath+"/overrides/liveness_nginx.sh", 0755)
+	if err != nil {
+		return errors.Wrap(err, "Could not set file permissions")
+	}
 	err = writer(util.NewTemplateWriter(nodeProbe, "nodereadiness", ReadinessLivenessSH),
 		"overrides", "readiness_node.sh")
 	if err != nil {
 		return err
 	}
+	err = os.Chmod(buildPath+"/overrides/readiness_node.sh", 0755)
```

Comment:
Var ikke helt lett å se at at det er 2 ulike systemer involvert her, node og nginx. Hadde det vært mulig å legge å separere i blokker som tar hånd om hhv node og nginx? Eventuelt refactoring for å samle felles funksjonalitet et sted?